### ISSUE TITLE
Design Review: Domain Request pages

### ISSUE THREAD

**Issue by abroddrick** [cisagov/manage.get.gov#2594]
### Issue description

Design should review and compare the output for the domain request pages and compare against the most recent figma and requirements.

In scope:
- Domain Request table
- Navigation
- All domain request pages visible to users
- Request form

Out of scope:
- Django Admin
- Other areas not related to domain request flow

### Acceptance criteria

- [ ] Design reviews the domain request pages described above
- [ ] Document any changes needed or new additions in a document, including with screenshots so it is easy to follow
- [ ] Discuss changes with dev and determine which are small email to be done as part of this ticket
- [ ] Work with dev to create new tickets for larger changes

### Additional context

**Working with dev:** Dev effort on this shouldn't be more than a "2" (2 full work days).

- Dev is responsible for answering questions from design, making minor changes (such as those that we would normally make on a PR).
- Anything that would take more than 4 hours to complete alone should be it's own ticket.
- If the amount of changes desired would result in the dev spending more than a full day on it, make a dev ticket for items where reasonable (communicate with Alysia).

NOTE: This design review does not include sending/testing email notifications to domain requesters. That will be tested separately once that work is implemented.

### Temporary Slack channel
Discussions for this design review should take place in this Slack channel: [2594-design-review](https://trussworks.slack.com/archives/C07TZN9MQ9Z)
### Links to other issues
-blocked by #2760 
- blocked by #2771 
- blocked by #2772 
- blocked by #2593 
- blocked by #2592 
- blocked by #2351
- blocked by #2595 
- blocked by #2596

**Comment by katypies** [comment]
I don't see why this is blocked by the ability to export reports. Can you help clarify, @abroddrick ?

**Comment by abroddrick** [comment]
@katypies  The only reason it was marked as blocked by the csv is because it is in the milestone. I am not opposed to us doing this review and then reviewing the csv separately. I'll let @Katherine-Osos and @witha-c have final say

**Comment by Katherine-Osos** [comment]
Agree, this shouldn't be blocked by csv downloads!

**Comment by vickyszuchin** [comment]
Moved all "In progress" sprint 57 tickets to the current sprint (sprint 58), per guidance.
